use t::Util;
use_ok "Rno::Exception::NotFoundResult";

done_testing;
